app/features-json/graph_ql/types/job_trigger_type.rb

```diff
@@ -0,0 +1,28 @@
+require "graphql"
+
+module FastlaneCI
+  # Definition of the different GraphQL types.
+  module Types
+    CommitJobTriggerType = GraphQL::ObjectType.define do
+      name "CommitJobTrigger"
+      field :branch, !types.String
+    end
+
+    NightlyJobTrigger = GraphQL::ObjectType.define do
+      name "NightlyJobTrigger"
+      field :branch, !types.String
+      field :hour, types.Int
+      field :minute, types.Int
```

Comment:
do these fields exist yet? If not, can we add them later?

Reply:
Do they exist! `NightlyJobTrigger` has those `attr_accessor`